VideoQuickStart/ViewController.swift

```diff
@@ -427,10 +442,15 @@ extension ViewController : RemoteParticipantDelegate {
         
         logMessage(messageText: "Unsubscribed from \(publication.trackName) video track for Participant \(participant.identity)")
 
-        if (self.remoteParticipant == participant) {
-            videoTrack.removeRenderer(self.remoteView!)
-            self.remoteView?.removeFromSuperview()
-            self.remoteView = nil
+        if self.remoteParticipant == participant {
+            cleanupRemoteParticipant()
+
+            // Find another Participant video to render, if possible.
+            if var remainingParticipants = room?.remoteParticipants,
```

Comment:
I think you need a `let` in there...

Reply:
It is compiling locally, so I think this is okay. We need `remainingParticipants` to be mutable.